test/lib/next-modes/base.ts

```diff
@@ -342,14 +343,8 @@ export class NextInstance {
   public async start(useDirArg: boolean = false): Promise<void> {}
   public async stop(): Promise<void> {
     this.isStopping = true
-    if (this.childProcess) {
-      let exitResolve
-      const exitPromise = new Promise((resolve) => {
-        exitResolve = resolve
-      })
-      this.childProcess.addListener('exit', () => {
-        exitResolve()
-      })
+    if (this.childProcess && !this.isStopping) {
+      const exitPromise = once(this.childProcess, 'exit')
```

Comment:
I had started making changes here and turned out they weren't needed, but I thought I'd keep this refactor since `once` seemed a little cleaner when I learned about it.